docs/source/user-guide/guide_data_dictionaries.rst

```diff
@@ -187,15 +194,14 @@ Creating a data dictionary via the UI
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 1. Log in as a user with the *Data: Create new content* permission.
 2. From the DKAN menu, select Data Dictionary -> Create.
-3. Enter a UUID for your data dictionary that will serve as its identifier.
-4. Enter a human readable title for your data dictionary.
-5. In the **Dictionary Fields** section, click the "Add one" button.
-6. Fill the form to define your field. Reference the Table Schema section above if needed.
-7. Repeat steps 5 and 6 for each field you want in your data dictionary.
-8. Click the "Save" button.
-9. See a list of your data dictionaries at `/api/1/metastore/schemas/data-dictionary/items/`
-10. Edit your data dictionary by going to `/admin/dkan/datasets`, then select "data-dictionary" from the data type filter, and click "Filter".
-11. Click the "Edit" link in the right-hand column.
+3. Enter a human readable title for your data dictionary.
+4. In the **Dictionary Fields** section, click the "Add one" button.
+5. Fill the form to define your field. Reference the Table Schema section above if needed.
+6. Repeat steps 5 and 6 for each field you want in your data dictionary.
```

Comment:
Think this should be "repeat steps 4 and 5".

Reply:
Yes! thank you